llvm/include/llvm/CodeGen/TargetLowering.h

```diff
@@ -625,6 +625,8 @@ class TargetLoweringBase {
     return isLoadBitCastBeneficial(StoreVT, BitcastVT, DAG, MMO);
   }
 
+  virtual bool isBitcastFree(EVT DstVT, EVT SrcVT) const { return false; }
```

Comment:
Is this still necessary?

Reply:
Its for multiuse case, since we are creating an additional instruction need it be free no?.

Reply:
Do you get instruction bloat without it? Vector bitcasts are typically free on most targets.

Reply:
I didn't notice any issue, the check just seemed basic due-diligence. I can make the default "true" if you think that makes more sense.

Reply:
I'd drop it entirely - we already have plenty of cases where we just bitcast between types.